src/k8s/requests/v2v/vmwareConfigMap.js

```diff
@@ -0,0 +1,40 @@
+import { VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMESPACES, VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAME } from './constants';
+import { ConfigMapModel } from '../../../models';
+
+const { info, warn } = console;
+
+const getVmwareConfigMapInNamespace = async ({ k8sGet, namespace }) => {
+  try {
+    return await k8sGet(ConfigMapModel, VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAME, namespace, null, {
+      disableHistory: true,
+    });
+  } catch (e) {
+    // This ConfigMap is expected to be created by a v2v operator which is currecntly under development.
+    // In the meantime, see https://github.com/kubevirt/web-ui-components/pull/507 for example.
+    info(
+      `The ${VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAME} can not be found in the ${namespace} namespace. Trying another in one ... Error: `,
+      e
+    );
+  }
+  return null;
+};
+
+export const getVmwareConfigMap = async props => {
+  // query namespaces sequentially to respect order
+  for (let index = 0; index < VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMESPACES.length; index++) {
```

Comment:
I get that the warns are different, but can we make at least this part generic? and add the comment from `getVmwareConfigMapInNamespace` to this function?